docs/docs/rest-api/public/api/v2/types/instance.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0 Library
+uses:
+  offer: offer.raml
+  strings: stringTypes.raml
+  unreachableStrategy: unreachableStrategy.raml
+
+types:
+  AgentInfo:
+    type: object
+    properties:
+      host: string
+      agentId?: string
+      region?: string
+      zone?: string
+      attributes: offer.AgentAttribute[]
+  InstanceState:
+    type: object
+    properties:
+      condition: strings.Condition
+      since: datetime
+      activeSince?: datetime
+      healthy?: boolean
+      goal: strings.Goal
+  Instance:
```

Comment:
The type is missing `tasksMap`, the instance id as an object and reservations. I do not think we should try too hard to be backwards compatible.

Reply:
I think it is fine though. `reservations` is optional, the `tasksMap` can be empty. so the only breaking change is the `instanceId: { idString: "..." }`.